corehq/motech/repeaters/views/repeat_records.py

```diff
@@ -186,27 +196,44 @@ def _payload_id_and_search_link(self, payload_id):
         )
 
     def _make_row(self, record):
+        checkbox = mark_safe(
+            """<input type="checkbox" onclick="uncheckSelects()" class="xform-checkbox"
```

Comment:
Set up the click handler in js instead of using an onclick handler here - this approach doesn't work well with requirejs, which we're in the process of migrating to.

Similar to the other checkboxes, overriding the value with an id isn't semantic. In this case, using a `data-id` attribute is probably a reasonable workaround.